<launch>

<arg name="cam" default="camera_1" />

<arg name="bag_file_path" default="$(find gige_cam_driver)/bagfiles/'$(arg cam)'.bag" />


  <node name="$(arg cam)_rosbag_record"
  pkg="rosbag" type="record" respawn="false" output="screen"
  args=" --duration=2 --output-name='$(arg bag_file_path)' /'$(arg cam)'/camera_info /'$(arg cam)'/image_raw" />
</launch>
